digits/model/views.py

```diff
@@ -202,9 +204,15 @@ def to_pretrained(job_id):
     snapshot_filename = None
     snapshot_filename = task.get_snapshot(epoch)
 
+    model_file = job.dir() + "/" + str(task.model_file)
+
+    # If jobs don't container model_file (too old), raise exception:
+    if not os.path.isfile(model_file):
+       raise werkzeug.exceptions.BadRequest('Model file not found in job dir. Job may be too old for conversion.')
```

Comment:
This is really disappointing, since everybody is going to try to convert an old job immediately. If it doesn't exist can we just write it? You need the "original" file, right? I think you can find it at `old_job.network`:
https://github.com/NVIDIA/DIGITS/blob/v4.0.0/digits/model/tasks/caffe_train.py#L231